lemur/plugins/lemur_gcp/auth.py

```diff
@@ -0,0 +1,31 @@
+import hvac
+import os
+
+from google.oauth2 import service_account
+from google.oauth2.credentials import Credentials
+
+
+def get_gcp_credentials(self, options):
```

Comment:
Do we still need so have "self" in the method definition given that this is outside of a Class context?

Reply:
We still need the arg but I renamed it to `plugin` so it's less misleading